Domains/4-Application/OpenSite/OpenSiteDraft.ecschema.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- ==================================================================================
+|  * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+|  * See LICENSE.md in the project root for license terms and full copyright notice.
+======================================================================================= -->
+<ECSchema schemaName="OpenSiteDrafts" alias="OpenSiteDrafts" version="01.00.00" xmlns="http://www.bentley.com/schemas/Bentley.ECXML.3.2" displayLabel="OpenSite Drafts" description="Drafts schema for drawing precise shapes in OpenSite+.">
+    <ECSchemaReference name="CoreCustomAttributes" version="01.00.04" alias="CoreCA" />
+    <ECSchemaReference name="BisCore" version="01.00.10" alias="bis" />
+    <ECSchemaReference name="RoadRailUnits" version="01.00.01" alias="rru" />
+
```

Comment:
Please add the following Custom-Attribute to advertise the status and layer of this schema:

```
   <ECCustomAttributes>
        <ProductionStatus xmlns="CoreCustomAttributes.01.00.03">
            <SupportedUse>NotForProduction</SupportedUse>
        </ProductionStatus>
        <SchemaLayerInfo xmlns="BisCustomAttributes.01.00.00">
            <Value>Application</Value>
        </SchemaLayerInfo>
    </ECCustomAttributes>
```
Note that this addition will force you to add BisCustomAttributes as a reference to this schema.

Reply:
Please look at the OpenSite schema as an example of how this Custom-Attribute is applied: https://github.com/iTwin/bis-schemas/blob/master/Domains/4-Application/OpenSite/OpenSite.ecschema.xml